### ISSUE TITLE
Small-sized bar chart not displaying correct bar heights

### ISSUE THREAD

**Issue by topex-psy** [imaNNeoFighT/fl_chart#200]
This issue occurs in my bar chart after upgrading fl_chart package from version: "0.6.2" to "0.8.1". Here's the screenshots:

Works perfectly in 0.6.2 | In 0.8.1 become problematic and overflowing, also all bars with diferent values have same heights
:-------------------------:|:--------------------------
![Screenshot_20200204-082846](https://user-images.githubusercontent.com/14989865/74280048-93c26580-4d4e-11ea-8dd5-ae8c152606bc.jpg) | ![Screenshot_20200212-041345](https://user-images.githubusercontent.com/14989865/74279797-2f9fa180-4d4e-11ea-85e3-ddcf273ba706.jpg)

With constraint height of 250, all bars has same height although they have different values | With constraint height of 550, it's okay
:-------------------------:|:--------------------------
![Screenshot_20200211-195621](https://user-images.githubusercontent.com/14989865/74280099-a9d02600-4d4e-11ea-8634-f00cf7b27f4f.jpg) | ![Screenshot_20200211-195610](https://user-images.githubusercontent.com/14989865/74280158-cc623f00-4d4e-11ea-8c15-997b7e3179e3.jpg)

Source code:
```
Card(
      shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(18)),
      color: Color(0xff81e5cd),
      child: Stack(
        children: <Widget>[
          Padding(
            padding: EdgeInsets.all(16),
            child: Column(
              crossAxisAlignment: CrossAxisAlignment.stretch,
              mainAxisAlignment: MainAxisAlignment.start,
              mainAxisSize: MainAxisSize.max,
              children: <Widget>[
                Text(
                  'Mingguan',
                  style: TextStyle(color: Color(0xff0f4a3c), fontSize: widget.params.mini ? 14 : 20, fontWeight: FontWeight.bold),
                ),
                SizedBox(height: 2,),
                Text(
                  'Grafik uang makan',
                  style: TextStyle(color: h.pickColor(Color(0xff379982), Color(0xff3a8775)), fontSize: widget.params.mini ? 12 : 16, fontWeight: FontWeight.bold),
                ),
                SizedBox(height: widget.params.mini ? 8 : 20,),
                Expanded(
                  child: Padding(
                    padding: EdgeInsets.symmetric(horizontal: widget.params.mini ? 8 : 12),
                    child: BarChart(mainBarData),
                  ),
                ),
              ],
            ),
          ),
        ],
      ),
    );
```

```
BarChartData get mainBarData => BarChartData(
    barTouchData: BarTouchData(
      touchTooltipData: BarTouchTooltipData(
        tooltipBgColor: Colors.blueGrey,
        getTooltipItem: (group, groupIndex, rod, rodIndex) {
          return BarTooltipItem("${f.dayName(_uangMakan[groupIndex].indeksHari)}\n${f.formatPrice(_uangMakan[groupIndex].nominal)}", TextStyle(color: Colors.white));
        }
      ),
      touchCallback: (barTouchResponse) {
        setState(() {
          if (barTouchResponse.spot != null &&
            barTouchResponse.touchInput is! FlPanEnd &&
            barTouchResponse.touchInput is! FlLongPressEnd) {
            _touchedIndex = barTouchResponse.spot.touchedBarGroupIndex;
          } else {
            _touchedIndex = -1;
          }
        });
      }
    ),
    gridData: FlGridData(
      show: true,
      drawHorizontalLine: !widget.params.mini,
      checkToShowHorizontalLine: (val) => val == _maksUangMakanPerHari / 1000,
      getDrawingHorizontalLine: (val) => FlLine(color: Colors.white.withOpacity(0.2), strokeWidth: 1,)
    ),
    titlesData: FlTitlesData(
      show: true,
      bottomTitles: SideTitles(
        showTitles: true,
        textStyle: TextStyle(color: Colors.white, fontFamily: "Lato", fontWeight: FontWeight.normal, fontSize: widget.params.mini ? 11 : 16),
        getTitles: (val) => _uangMakan[val.toInt()] == null ? val.toString() : f.dayAbbr(_uangMakan[val.toInt()].indeksHari),
        margin: widget.params.mini ? 4 : 8,
      ),
      leftTitles: widget.params.mini ? SideTitles(showTitles: false,) : SideTitles(
        showTitles: true,
        getTitles: (val) => val == _maksUangMakanPerHari / 1000 ? "${val.toInt()}K" : null,
        textStyle: TextStyle(color: Colors.white, fontSize: 14),
        margin: 8
      ),
    ),
    borderData: FlBorderData(show: false,),
    barGroups: _barGroups
  );
```

**Comment by imaNNeoFighT** [comment]
Hi, Please prepare a complete reproducible code (with main function)

**Comment by topex-psy** [comment]
I have it in my private published project, so I can't give any full reproducible code for now, but surely I will make a dummy repo for this issue as simple as I can. I definitely try to help until this issue fixed because I need my app to run smoothly for my users too.

**Comment by imaNNeoFighT** [comment]
We should fix this issue, and it's our main concern.
Without a full reproducible code, we can't find the problem easily,
then please make a dummy project to reproduce it.
Thanks for reporting!

**Comment by topex-psy** [comment]
I have published the simple reproduction of the issue here: https://github.com/topex-psy/fl_chart_bug
@imaNNeoFighT

**Comment by topex-psy** [comment]
Do you currently working for this issue? Any update?

**Comment by imaNNeoFighT** [comment]
Fixed in [0.8.3](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.8.3), sorry for the delay, and Thanks for reporting!
